packages/backend/src/tokens/interstitial.test.ts

```diff
@@ -0,0 +1,38 @@
+import type QUnit from 'qunit';
+import sinon from 'sinon';
+
+import runtime from '../runtime';
+import { loadInterstitialFromBAPI } from './interstitial';
+
+export default (QUnit: QUnit) => {
+  const { module, test } = QUnit;
+
+  module('loadInterstitialFromBAPI', hooks => {
+    let fakeFetch;
+
+    hooks.afterEach(() => {
+      fakeFetch?.restore();
+    });
+
+    test('executes GET request to /v1/public/interstitial', async assert => {
+      fakeFetch = sinon.stub(runtime, 'fetch');
+      fakeFetch.onCall(0).returns({
+        text: () => '<html>Interstitial Page</html>',
+        ok: true,
+        status: 200,
+      });
+
+      const publishableKey = 'pk_test_Y2xlcmsuaW5jbHVkZWQua2F0eWRpZC05Mi5sY2wuZGV2JA';
+      await loadInterstitialFromBAPI({
+        apiUrl: 'https://api.clerk.test',
+        publishableKey,
+        frontendApi: 'accounts.included.katydid-92.lcl.dev',
+      });
```

Comment:
Maybe pointless with the mocks, but do we want to assert on the fetch response here as well? I see that `loadInterstitialFromBAPI()` returns the interstitial response text.